host/app/components/card-editor.gts

```diff
@@ -97,7 +97,10 @@ export default class Preview extends Component<Signature> {
     super(owner, args);
     if (this.args.card.type === 'existing') {
       taskFor(this.loadData).perform(this.args.card.url);
-      this.interval = setInterval(() => taskFor(this.loadData).perform((this.args.card as any).url), 1000);
+      // This polling seems to be exposing some kind of race condition in our
+      // index--the data.meta.lastModified is undefined after PATCHing for some
+      // reason even though this value definitely exists in our search index.
+      // this.interval = setInterval(() => taskFor(this.loadData).perform((this.args.card as any).url), 1000);
```

Comment:
I ran across this bug when I tried to update a card instance. I didn't get a chance to look into it more deeply, but definitely we should look into this--it looks like a really interesting bug